tests/e2e/utils/utils.go

```diff
@@ -0,0 +1,148 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/apimachinery/pkg/util/wait"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+const (
+	DeleteNSTimeout = 10 * time.Minute
+
+	SingleCallTimeout = 5 * time.Minute
+
+	// How often to Poll pods, nodes and claims.
+	Poll = 2 * time.Second
+
+	pollShortTimeout = 1 * time.Minute
+	pollLongTimeout  = 5 * time.Minute
+
+	PodImage = "k8s.gcr.io/pause:3.1"
+)
+
+func findExistingKubeConfig() string {
+	defaultKubeConfig := "/etc/kubernetes/admin.conf"
```

Comment:
Can just use default rule, this file is not likely to exist

Reply:
It refers to defaultKubeConfig and findExistingKubeConfig in https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/token.go